apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/nfts/components/nft-property.tsx

```diff
@@ -8,22 +7,19 @@ interface NftPropertyProps {
 
 export const NftProperty: React.FC<NftPropertyProps> = ({ property }) => {
   return (
-    <Card as={Flex} flexDir="column" gap={2}>
-      {property?.trait_type && (
-        <Text
-          size="label.sm"
-          color="primary.500"
-          textAlign="center"
-          lineHeight={1.2}
-        >
-          {property?.trait_type}
-        </Text>
-      )}
-      <Text size="label.md" textAlign="center">
-        {typeof property?.value === "object"
-          ? JSON.stringify(property?.value || {})
-          : property?.value}
-      </Text>
+    <Card className="flex p-3">
+      <div className="m-auto flex flex-col items-center">
+        {property?.trait_type && (
+          <p className="text-center text-primary text-sm leading-[1.2]">
+            {property?.trait_type}
+          </p>
+        )}
+        <p className="text-center">
```

Comment:
`Text` usually renders the equivalent of `text-sm text-muted-foreground` - please verify the rendered text

Reply:
updated as suggested